src/Utils/WaterMod.jl

```diff
@@ -1,10 +1,21 @@
-module WaterVaporMod
-using PhysCon
+module WaterMod
+
+using ..PhysCon
 #-----------------------------------------------------------------------
 # DESCRIPTION:
 # Calculate saturation vapor pressure and latent heat of vaporization
 #
-export SatVap
+export Lv, SatVap, μ_l
```

Comment:
It would be good to move to using `MoistThermodynamics` for this--as above, to ensure energetic and thermodynamic consistency across model components. Perhaps also separate PR.

Reply:
I agree, we should try to not re-define constants, etc. Is there a way to embed this in Land but still being able to "run" Land even without having all of CLIMA?

Reply:
My understanding is yes (you can import modules selectively).@charleskawczynski can help with it.

Reply:
So, there is a Moist Thermo package, but CLIMA still depends on an internal version. We could continue with peeling it off if that's the right direction. The other option would be to make this package depend on CLIMA, but CLIMA does have some heavy dependencies.

Reply:
Would be good not to have it depend on CLIMA.

Reply:
We'll need `MoistThermodynamics` in land, ocean, sea ice etc.. Ditto for CliMA Parameters. So we'll need some way to run some of these models as they are being developed, even when they do not use DG methods etc. Can you propose a solution?

Reply:
From what I understand, there's two options:

 - Make this package depend on CLIMA
    - pros: MoistThermo can depend on CLIMA
    - cons: brings in all of the dependencies
 - Finish splitting moist thermo off into its own package (basically done, but there some updates in CLIMA that need to be moved to the moist thermo package) and make CLIMA depend on it (not done, yet)
    - pros: Much lighter set of dependencies
    - cons: Migrates from mono-repo design

Reply:
What we’ll need for all model components is access to the tools for CLIMAParameters, free parameters, moist thermodynamics, and calibrate-emulate-sample. Are all of these separate packages besides moist thermo? If so, then we should split off moist thermo. If not, we need a straightforward solution for using all of these. Maybe making everything CLIMA dependent is an option then, but it’ll be unattractive for some because code like this here will also be used outside CLIMA.

Reply:
Yes, I believe so. Sounds like a good idea. I’ll start the transfer.

Reply:
Yes, I believe so. Sounds like a good idea. I’ll start the transfer.